src/deploy/index.js

```diff
@@ -52,13 +52,18 @@ module.exports = async (api, siteId, dir, opts) => {
     hashFns(fnDir, opts)
   ])
 
-  statusCb({
-    type: 'hashing',
-    msg:
-      `Finished hashing ${Object.keys(files).length} files` +
-      (fnDir ? ` and ${Object.keys(functions).length} functions` : ''),
-    phase: 'stop'
-  })
+   const filesCount = Object.keys(files).length
+   const functionsCount = Object.keys(functions).length
+
+   statusCb({
+     type: 'hashing',
+     msg: `Finished hashing ${filesCount} files` + (fnDir ? ` and ${functionsCount} functions` : ''),
+     phase: 'stop'
+   })
+
+   if (filesCount <= 0 && functionsCount <= 0) {
```

Comment:
[sand] Since `filesCount` and `functionsCount` cannot be negative, would `===` be clearer than `<=`?

Reply:
That's me being probably too protective with my code.